src/component/Body/Weather/index.tsx

```diff
@@ -46,8 +49,8 @@ const Weather: React.FC<WeatherProps> = (props) => {
 
   //process before render
   if (!props.isLoading && props.weatherData) {
-    const hourly = props.weatherData[1].hourly[4];
-    const Dailies = props.weatherData[1].daily.slice(0, 3);
+    const hourly = props.weatherData.AllDate.hourly[4];
+    const Dailies = props.weatherData.AllDate.daily.slice(0, 3);
```

Comment:
I would call this hell of dot, the access path is too long. Why is destructure still not here?